### PR TITLE
Adds Juiceboxes/Cartons + Apple juice & Choc. Milk

### PR DESCRIPTION
**Pull request by TehZombehz**
:cl:
add: Small cartons can be crafted using 1 piece of cardboard. Don't be late for school.
add: Apples can now be juiced for apple juice. Chocolate milk can now be crafted using milk and cocoa.
tweak: Chocolate bar recipe has been modified to compensate for chocolate milk. The soy milk version of the chocolate bar recipe remains unchanged.
bugfix: Grapes can now be properly juiced for grape juice using a grinder.
/:cl:

juice

Adds craftable small cartons and juice boxes. Sprites by Nienhaus.
Volume of small cartons is 15 units.
Also adds the ability to juice apples for apple juice.
Adds chocolate milk. Changes standard chocolate bar recipe to compensate (soy milk version of the chocolate bar recipe remains the same).
Fixes being unable to juice grapes in a grinder.

### CODE REVIEW

**Comment by Cheridan** [comment]
Pfft this is perfect

**Comment by Tacolizard** [comment]
powercreep. these should require at least three cardboard. easily available liquid holders are already too prevalent

File: code/modules/food_and_drinks/drinks/drinks.dm [hunk 1]
```diff
@@ -216,6 +216,32 @@
...
+	desc = "A small carton, intended for holding drinks."
+	icon_state = "juicebox"
+	volume = 25
+
+/obj/item/weapon/reagent_containers/food/drinks/sillycup/smallcarton/on_reagent_change()
+	if (reagents.reagent_list.len > 0)
+		switch(reagents.get_master_reagent_id())
+			if("orangejuice")
+				icon_state = "juicebox"
+				name = "orange juice box	"
```
Comment (Iamgoofball) [inline code/modules/food_and_drinks/drinks/drinks.dm]: Forgot "

**Review by Iamgoofball — CHANGES_REQUESTED** [review]

**Comment by Bawhoppen** [comment]
Needs to be smaller, why would this tiny juicebox hold 35% volume of a bucket

File: code/modules/food_and_drinks/drinks/drinks.dm [hunk 2]
```diff
@@ -216,6 +216,32 @@
...
 		icon_state = "water_cup_e"
 
+/obj/item/weapon/reagent_containers/food/drinks/sillycup/smallcarton
+	name = "small carton"
+	desc = "A small carton, intended for holding drinks."
+	icon_state = "juicebox"
+	volume = 25
+
+/obj/item/weapon/reagent_containers/food/drinks/sillycup/smallcarton/on_reagent_change()
+	if (reagents.reagent_list.len > 0)
```
Comment (Jordie0608) [inline code/modules/food_and_drinks/drinks/drinks.dm]: This can just be `if(reagents.reagent_list.len)`

**Comment by nullbear** [comment]
I agree. Make em smaller, but reduce cost. 

Cola cans shouldn't really hold as much as they do either, but whatever. Despite cardboard being "all access" it's often rare to see someone with lots at a time.

Oh and I'm still dead. Don't bug me.

**Comment by Anonmare** [comment]
If they can't spill their contents, have a small reagent cap and can't be used to make IEDs or anything like that then they should be good.

**Comment by Nienhaus** [comment]
@TehZombehz what other juices you want?

**Comment by TehZombehz** [comment]
@Nienhaus Distinct orange, apple (though the default juice box already looks like apple), and grape juice cartons.
These can spill their contents since they are just children of paper cups. Super tiny paper cups can already hold 10u by default, which is 1/5 of a bucket. I'm not against reducing the size and cost, though. How does 15u for 1 sheet sound?

**Comment by nullbear** [comment]
15u for 1 sheet is great.

**Comment by jughu** [comment]
Atleast cardboards are limited soda cans are not just rebuild the vending machine and you get a new supply! And they hold more

**Comment by Nienhaus** [comment]
@TehZombehz https://www.dropbox.com/s/s817zedrhz9xw4t/Misc%20drinks.dmi?dl=0 Here you go.

**Comment by optimumtact** [comment]
Now I can spill chocolate milk down the sink

**Pull request merged**